src/e2e/java/teammates/e2e/pageobjects/AdminSearchPage.java

```diff
@@ -205,7 +205,7 @@ public void resetStudentGoogleId(StudentAttributes student) {
     public WebElement getInstructorRow(InstructorAttributes instructor) {
         String xpath = String.format(
                 "//table[@id='search-table-instructor']/tbody/tr[td[%d][span[text()='%s']] and td[%d]='%s']",
-                INSTRUCTOR_COL_COURSE_ID, instructor.getCourseId(), INSTRUCTOR_COL_NAME, instructor.getName());
+                INSTRUCTOR_COL_COURSE_ID, " " + instructor.getCourseId() + " ", INSTRUCTOR_COL_NAME, instructor.getName());
```

Comment:
@samuelfangjw Honestly I'm a beginner in fronend development and I just notice this thing, which seems very weird to me. So if I write 

> \<span>text\</span>

, the text I see in Chrome Inspect is just "text". However, if I write

> \<span>
> text
> \</span>

, the text I see in Chrome Inspect is " text " (spaces at the front and end).

Is this actually a very normal behavior and common knowledge?

Reply:
Actually it's supposed to be a common knowledge probably. So new line(s) will introduce a space. 🤔

Reply:
Based on our UI conventions, tool-tipped text should be indicated with a dashed underline.

Reply:
> Based on our UI conventions, tool-tipped text should be indicated with a dashed underline.

Hi prof, you mean the course name should have a dashed underline? Actually I didn't change the behavior of that, so the course name tooltip is essentially how it looked like before; but I think I'm able to add a dashed underline there.

Reply:
This looks a little ugly to me.
![image](https://user-images.githubusercontent.com/68739475/169444122-356779f8-7586-4deb-bd01-3e218b1f517e.png)